recipe/meta.yaml

```
@@ -7,12 +7,12 @@ package:
   version: {{ version|replace("-", "_") }}
 
 source:
-  url: https://github.com/EDIorg/EMLassemblyline/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 1596d2b2dc00049103746d9cd563cbc8f36e5b76991c9305b8b2d2905fbfb924
+  url: https://github.com/yvanlebras/EMLassemblyline/archive/refs/heads/{{ version }}-dev.zip
```

This is not robust. We need a proper release or a tag, not merely the latest head when recipe runs. Otherwise, rerunning the build gives different results depending on the state of the upstream repository.

Please ask upstream to create a Release. Better, get this merged rather than switching to a Fork.

Even if there is a good reason for this, the original should be retained so we can easily transition back to the official upstream:

```suggestion
  url:
    - https://github.com/EDIorg/EMLassemblyline/archive/refs/tags/v{{ version }}.tar.gz
    - https://github.com/yvanlebras/EMLassemblyline/archive/refs/heads/{{ version }}.zip
```

---

Understand. I created a dedicated "dev" tag and make a PR to propose update the code https://github.com/PaulineSGN/r-emlassemblyline-feedstock/pull/1